cmd/ocm/gcp/delete-wif-config.go

```diff
@@ -80,6 +80,13 @@ func deleteWorkloadIdentityConfigurationCmd(cmd *cobra.Command, argv []string) e
 		return nil
 	}
 
+	// Check if wif-config can be deleted with dry-run
+	_, err = connection.ClustersMgmt().V1().GCP().WifConfigs().
+		WifConfig(wifConfig.ID()).Delete().DryRun(true).Send()
+	if err != nil {
+		return errors.Wrapf(err, "failed to delete wif config %q", wifConfig.ID())
+	}
+
```

Comment:
This check should occur prior to the servicing of cli's dry-run. We don't want the user to generate the cleanup script if the cluster is in use.